<launch>
    <node pkg="pepwave_gps" type="gps.py" name="publish_gps" output="screen">
        <param name="ip_address" value="192.168.50.1" />
        <param name="port" value="60660" />
        <param name="gps_frame" value="ugr/car_base_link/gps" />
        <param name="use_gps_time" value="False" />
    </node>
</launch>